pkg/controller/testutil/test_utils.go

```diff
@@ -152,7 +152,7 @@ func (m *FakeNodeHandler) Get(ctx context.Context, name string, opts metav1.GetO
 			return &nodeCopy, nil
 		}
 	}
-	return nil, nil
+	return nil, fmt.Errorf("node %s does not exist", name)
```

Comment:
Use NewNotFound https://github.com/kubernetes/apimachinery/blob/master/pkg/api/errors/errors.go#L144

Reply:
Updated. Thanks for your review.